### PR TITLE
chore: remove dependency on sdk-core from examples

### PR DESCRIPTION
**Pull request by pratik151192**
Added more exports to `index.ts` to remove examples dependency on `sdk-core` after a release. Tested with local installation that was failing on the first PR and now its succeeding after these imports.

### CODE REVIEW

**Comment by pratik151192** [comment]
Hmm looks like this was added for node version 14 to work

```
Error:    8:1  error  '@gomomento/sdk-core' should be listed in the project's dependencies. Run 'npm i -S @gomomento/sdk-core' to add it  import/no-extraneous-dependencies
Error:   16:8  error  "@gomomento/sdk-core" is extraneous                                                                                 node/no-extraneous-import
```

**Comment by cprice404** [comment]
> Hmm looks like this was added for node version 14 to work
> 
> ```
> Error:    8:1  error  '@gomomento/sdk-core' should be listed in the project's dependencies. Run 'npm i -S @gomomento/sdk-core' to add it  import/no-extraneous-dependencies
> Error:   16:8  error  "@gomomento/sdk-core" is extraneous                                                                                 node/no-extraneous-import
> ```

I believe that error is telling you about another place where core is explicitly used in an import, I don't think it has to do with the node version. We should fix the imports.

**Review by cprice404 — COMMENTED** [review]
more imports we need to fix I think

**Comment by pratik151192** [comment]
> I believe that error is telling you about another place where core is explicitly used in an import, I don't think it has to do with the node version. We should fix the imports.

ah yeah will fix

**Review by bruuuuuuuce — COMMENTED** [review]
we should also update the package.json and remove the dep

**Comment by pratik151192** [comment]
@bruuuuuuuce 

> we should also update the package.json and remove the dep

yep i can only successfully do that after a release with the changes in this PR

File: packages/client-sdk-nodejs/src/index.ts [hunk 1]
```diff
@@ -143,6 +143,8 @@ import {
   TopicSelector,
   AllCaches,
   AllTopics,
+  LeaderboardOrder,
+  ILeaderboard,
```
Comment (cprice404) [inline packages/client-sdk-nodejs/src/index.ts:147]: thank you! do we need to do this in the web sdk too?

**Review by cprice404 — APPROVED** [review]

**Review by cprice404 — APPROVED** [review]

File: packages/client-sdk-nodejs/src/index.ts (hunk 1 above)
Comment (pratik151192) [reply packages/client-sdk-nodejs/src/index.ts:147]: yep did that and mistakenly committed my local `npm i` installation.. will update examples once released!

**Pull request merged**